conans/client/tools/apple.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import subprocess
+
+
+def cmd_output(command):
+    return subprocess.check_output(command).decode().strip()
+
+
+def is_apple_os(_os):
+    """returns True if OS is Apple one (Macos, iOS, watchOS or tvOS"""
+    return str(_os) in ['Macos', 'iOS', 'watchOS', 'tvOS']
+
+
+def to_apple_arch(arch):
+    """converts conan-style architecture into Apple-style arch"""
+    return {'x86': 'i386',
+            'x86_64': 'x86_64',
+            'armv7': 'armv7',
+            'armv8': 'arm64',
+            'armv7s': 'armv7s',
+            'armv7k': 'armv7k'}.get(str(arch))
+
+
+def apple_sdk_name(settings):
```

Comment:
sdk name for what? CMake?

Reply:
for xcrun (-sdk option)
rename to something like apple_xcrun_sdk_name?